perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -44,6 +44,15 @@ chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 sudo umount gcs
 
+# Mounting gcsfuse for running integration tests for the mountedDirectory flag
```

Comment:
for pr_perf_test why do you need to run integration tests on master branch. perf tests are run to do comparision in the perf numbers. only any pr running integration tests on pr branch should be sufficient.

Reply:
fixed